mlflow/pipelines/steps/train.py

```diff
@@ -457,6 +489,34 @@ def render_schema(inputs, title):
         else:
             run_card_tab.add_markdown("MODEL_URI", f"**MLflow Model URI:** `{model_uri}`")
 
+        # Tab 8: Best Parameters
+        if best_estimator_params:
+            tuning_params_card_tab = card.add_tab(
+                "Best Parameters",
+                "{{ SEARCH_SPACE }} " + "{{ BEST_PARAMETERS }} ",
+            )
+            tuning_params = yaml.dump(self.step_config["tuning"]["parameters"])
+            tuning_params_card_tab.add_html(
+                "SEARCH_SPACE",
+                f"<b>Tuning search space:</b> <br><pre>{tuning_params}</pre><br><br>",
+            )
+            best_parameters_yaml = os.path.join(output_directory, "best_parameters.yaml")
+            if os.path.exists(best_parameters_yaml):
+                best_hardcoded_parameters = open(best_parameters_yaml).read()
+                tuning_params_card_tab.add_html(
+                    "BEST_PARAMETERS",
+                    f"<b>Best parameters:</b><br>"
+                    f"<pre>{best_hardcoded_parameters}</pre><br><br>",
+                )
+
+        # Tab 9: HP trials
+        if tuning_df is not None:
```

Comment:
@dbczumar i think you had a comment on a previous version that this tab should be conditioned on `best_estimator_params`. the creation of `tuning_df` on line 232 only happens if `best_estimator_params` exists, so i believe i am reflecting that behavior

Reply:
Agreed! Nice!